captcha/events.py

```diff
@@ -60,3 +60,25 @@ async def on_member_join(self, member: Member):
     @commands.Cog.listener()
     async def on_member_remove(self, member: Member):
         await self.cleaner(member)
+
+    @commands.is_owner()
```

Comment:
I think you're wanting `@commands.guildowner()` here.

```suggestion
    @commands.guildowner()
```

Reply:
Oh yes. I had thought for the bot owner only, but the server owner should be able to do it.